doc/release/README.multilocale.rst

```diff
@@ -0,0 +1,233 @@
+============================
+Multilocale Chapel Execution
+============================
+
+To run Chapel on multiple locales, the following steps are required:
+
+
+0. The following information describes a fairly generic use of
```

Comment:
Any reason to not use auto-numbering here?  I think it's ok if we don't start at 0. 

Also, the auto-numbering is amazing! It used to such a pain to add steps to this file